components/eoy_artist_list/_artist_item.jade

```diff
@@ -1,6 +1,6 @@
 .eoy-artist-item
   .eoy-artist-item__thumbnail
-    a(href="/artist/#{artist.slug}"): img(src="#{sd.API_URL}/api/v1/artist/#{artist.slug}/image")
+    a(href="/artist/#{artist.slug}"): img(src="/search/image/artist/#{artist.slug}")
```

Comment:
I was only wondering if we have control over the version, e.g. fetch a smaller one, but that doesn’t seem to be the case, right?